pkg/gce-pd-csi-driver/gce-pd-driver.go

```diff
@@ -73,6 +73,8 @@ func (gceDriver *GCEDriver) SetupGCEDriver(name, vendorVersion string, extraVolu
 		csi.ControllerServiceCapability_RPC_LIST_VOLUMES,
 		csi.ControllerServiceCapability_RPC_LIST_VOLUMES_PUBLISHED_NODES,
 		csi.ControllerServiceCapability_RPC_CLONE_VOLUME,
+		// TODO: Enable once sanity tests are upgraded to csi-test v5.3.0
+		// csi.ControllerServiceCapability_RPC_MODIFY_VOLUME,
```

Comment:
This shouldn't be commented out. This is needed for this feature to work.

Reply:
Done